### DISCUSSION TITLE
Javascript events not collected in the events tab

### DISCUSSION THREAD

**Discussion by eguiluz** [wire-elements/wire-spy#22] [Feature Requests]
Events emitted with javascript from blade views are not recorded in the events tab.

```
<script type="module">
    document.addEventListener('livewire:initialized', () => {
        Livewire.dispatch('eventNameHere', eventDataHere);
    });
</script>
```

The event eventNameHere (which is picked up in the component correctly with #[On('eventNameHere')] is not registered by wire-spy.

**Comment by PhiloNL** [comment]
Hi @eguiluz,
At the moment only server-side triggered events are recorded. I'll add this to the feature request list :)

**Reply by eguiluz** [reply]
Great! I'll be keeping an eye on the thread and updates :D
